.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: objective-c
+osx_image: xcode7.2
+branches:
+  only: master
+script: placeholder # workaround for https://github.com/travis-ci/travis-ci/issues/4681
+matrix:
+  include:
+    - script: ./build.sh verify-swiftlint
+      env: JOB=verify-swiftlint
+      before_install: brew install swiftlint
+    - script: ./build.sh verify-docs
+      env: JOB=verify-docs
+      before_install: gem install jazzy
```

Comment:
We might be able to save some additional time by making use of [Travis' optional caching logic for bundler](https://docs.travis-ci.com/user/caching/#Enabling-Bundler-caching).